src/main/java/com/whalespottingjava/controllers/AdminController.java

```diff
@@ -26,4 +32,23 @@ public String getAdminPage(Model model) {
         model.addAttribute("sightings", sightingService.getAllPendingSightings());
         return "admin";
     }
+
+    @PostMapping("/admin")
+    @ResponseStatus(HttpStatus.CREATED)
+    public String postSightings(@RequestBody List<AdminApprovalRequest> list, Model model) {
+        for (AdminApprovalRequest sighting: list) {
+            if (sighting.getApproved() == false) {
+                sightingService.deleteRejectedPendingSighting(sighting.getId());
+            }
+            if (sighting.getApproved() == true) {
+                Sighting currentSighting = sightingService.getSightingById(sighting.getId());
+                sightingService.addSighting(currentSighting);
+            }
+            if (sighting.getApproved() == null) {
```

Comment:
Do we get this behaviour implicitly anyways if we leave it out? In that case we can leave it in as an explicit case, but I would probably remove this case